lblod/app-digitaal-loket
# Remove English requirement when YourEurope is not chosen

(Addresses DL-3983) Removes the English translation requirements when `YourEurope` is not chosen as the publication channel in the `Eigenschappen` tab in a LPDC form.

This is to be tested in tandem with the following PR for `lpdc-management-service`: https://github.com/lblod/lpdc-management-service/pull/32.

To test:
* Log in as any Gemeente
* Go to the `Producten- en dienstencatalogus` module
* Add an existing public service or create a completely new one
* Toggle the publication channel fields in the `Eigenschappen` tab between empty and `YourEurope`
* The sections listed below should switch from `Verplicht` (when `YourEurope` is chosen) to nothing (when `YourEurope` is not chosen)

If it is chosen, `Titel`, `Beschrijving`, `Voorwaarden`, `Bewijsstukken` (Nested inside `Voorwaarden`), `Procedure`, `Website procedure` (Nested inside `Procedure`), `Kosten`, `Financiële Voordelen`, `Meer info` sections should have `Verplicht` alongside the English title and description fields.
